### PR TITLE
Geany rebuild

### PR DESCRIPTION
**Pull request by satmandu**
- Rebuild, (recent startup speed fix & gtk2 cleanup)
- modify postinstall to make sure geany icons get properly registered.

Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] i686
- [x] armv7l

### CODE REVIEW

File: packages/geany.rb [hunk 1]
```diff
@@ -41,12 +46,16 @@ def self.install
   end
 
   def self.postinstall
+    # This is needed to aoid "Error loading theme icon 'geany-build' for stock" messages.
```
Comment (uberhacker) [inline packages/geany.rb:49]: avoid?  Missing v?  What package provides `gtk-update-icon-cache`?  If it is a dependency then no need to check if the binary exists first.

File: packages/geany.rb [hunk 2]
```diff
@@ -41,12 +46,16 @@ def self.install
...
+    end
     puts
     puts "To get started, type 'geany'.".lightblue
     puts
-    puts "To completely uninstall, execute the following:".lightblue
-    puts "crew remove geany".lightblue
-    puts "rm -rf ~/.config/geany".lightblue
+    puts 'To completely uninstall, execute the following:'.lightblue
+    puts 'crew remove geany'.lightblue
+    puts 'rm -rf ~/.config/geany'.lightblue
```
Comment (uberhacker) [inline packages/geany.rb:58]: Could these steps be added to `def self.remove` with a prompt instead?

File: packages/geany.rb (hunk 1 above)
Comment (satmandu) [reply packages/geany.rb:49]: True too. I think gtk3 has that, so just adding the dependency should take care of it. And yes that's a typo.

File: packages/geany.rb (hunk 2 above)
Comment (satmandu) [reply packages/geany.rb:58]: Yes, that could be handled in such a manner. If I get to it before I leave tomorrow I will add that.

File: packages/geany.rb (hunk 1 above)
Comment (satmandu) [reply packages/geany.rb]: Looks like gtk3 & gtk2 have it, so this is already covered...

File: packages/geany.rb (hunk 2 above)
Comment (satmandu) [reply packages/geany.rb]: Added self.remove

File: packages/geany.rb [hunk 3]
```diff
@@ -3,32 +3,40 @@
...
   })
 
-    depends_on 'gtk3'
-    depends_on 'xdg_base'
-    depends_on 'sommelier'
+  depends_on 'gtk3'
+  depends_on 'xdg_base'
+  depends_on 'sommelier'
 
+  @xdg_config_home = ENV['XDG_CONFIG_HOME']
```
Comment (uberhacker) [inline packages/geany.rb:29]: Doesn't this make an assumption the environment variable is set?  What if it isn't set?

File: packages/geany.rb [hunk 4]
```diff
@@ -3,32 +3,40 @@
...
+  @xdg_config_home = "#{CREW_PREFIX}/.config" if @xdg_config_home.to_s.empty?
```
Comment (uberhacker) [inline packages/geany.rb:30]: Never mind.  I see you set it here is it's empty.

**Pull request merged**